### PR TITLE
[Docs] Add why Vizro-AI section

### PR DESCRIPTION
**Pull request by nadijagraca**
## Notice

- [ ] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Why use Vizro-AI over ChatGPT?
+
+Vizro-AI offers an easy way to turn natural language instructions into professional visualisations, like interactive charts and dashboards, without needing advanced coding skills.
+
+Similar to ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
```
Comment (stichbury) [inline vizro-ai/docs/pages/explanation/why-use-vizro-ai.md:5]:
```suggestion
Like ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English, or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
```

File: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+- **Tailored for data visualisation**: Vizro-AI is specifically designed to create and design charts and dashboards.
+- **Enforced design best practices automatically**: Using Vizro’s themes, your visualizations are built with best practices in mind from the start.
```
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-ai/docs/pages/explanation/why-use-vizro-ai.md:8]: I would steer away from "enforced" since that sounds a bit draconian. Maybe "Design best practices applied automatically" ?

File: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- **Instant chart execution**: Vizro-AI generates and executes chart code automatically, allowing you to see the chart output immediately
```
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-ai/docs/pages/explanation/why-use-vizro-ai.md:9]: Please replace "allowing" with "enabling"

**Review by stichbury — APPROVED** [review]
LGTM 🌟 🌟 🌟 🌟 🌟

File: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Why use Vizro-AI over ChatGPT?
+
+Vizro-AI offers an easy way to turn natural language instructions into professional visualisations, like interactive charts and dashboards, without needing advanced coding skills.
+
+Similar to ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
+
+- **Tailored for data visualisation**: Vizro-AI is specifically designed to create and design charts and dashboards.
```
[the remark this answers is not in the document]
Comment (maxschulz-COL) [reply vizro-ai/docs/pages/explanation/why-use-vizro-ai.md:7]: I like those suggestions - I think they might both become individual points even

File: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Why use Vizro-AI over ChatGPT?
+
+Vizro-AI offers an easy way to turn natural language instructions into professional visualizations, like interactive charts and dashboards, without needing advanced coding skills.
+
+Like ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English, or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
+
+- **Tailored for data visualization**: Vizro-AI is specifically designed to create and design charts and dashboards.
+- **Enforced design best practices automatically**: Using Vizro’s themes, your visualizations are built with best practices in mind from the start.
```
Comment (huong-li-nguyen) [inline vizro-ai/docs/pages/explanation/why-use-vizro-ai.md:8]:
```suggestion
- **Data visualization best practices applied**: Vizro’s chart theme ensures your charts follow many best practices, such as using color-blind friendly colors, maintaining a decluttered design, and automatically placing headers and footers.
```

File: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Why use Vizro-AI over ChatGPT?
+
+Vizro-AI offers an easy way to turn natural language instructions into professional visualizations, like interactive charts and dashboards, without needing advanced coding skills.
+
+Like ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English, or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
+
+- **Tailored for data visualization**: Vizro-AI is specifically designed to create and design charts and dashboards.
```
Comment (huong-li-nguyen) [inline vizro-ai/docs/pages/explanation/why-use-vizro-ai.md:7]:
```suggestion
- **Tailored for data visualization**: Vizro-AI is specifically designed to create charts and dashboards with in-built beautiful visual design best practices.
```

I would rephrase to something like the above because Vizro-AI and Vizro overall don't assist in designing new charts and dashboards. Instead, they provide an opinionated design through Vizro, but they don't "help" in the sense of creating new designs, which I would partially understand from above

File: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+- **Enforced design best practices automatically**: Using Vizro’s themes, your visualizations are built with best practices in mind from the start.
+- **Prompt optimization**: Vizro-AI uses advanced prompt engineering to incorporate additional context and data-specific details, making your visualizations more accurate.
```
Comment (huong-li-nguyen) [inline vizro-ai/docs/pages/explanation/why-use-vizro-ai.md:9]: This seems too focused on the technical implementation rather than the user benefit. As a user, I'm more interested in how it provides data insights that complement my charts. So I think the emphasis should be on the added value rather than the technical details like prompt optimization?  Maybe something like below?

```suggestion
- **Data Insights**: Vizro-AI leverages advanced prompt engineering to integrate additional context and data-specific insights, enriching your visualizations with more comprehensive information and allowing for more effective data storytelling.
```

**Review by huong-li-nguyen — APPROVED** [review]
